### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by guenaelthiriet** [hypnoglow/helm-s3#67]
Hi,

I was wondering if there were any plans for windows support?
When installing the plugin under windows (git bash, in admin mode) I am getting:

Downloading and installing helm-s3 v0.8.0 ...
Unsupported OS type
helm-s3 install hook failed. Please remove the plugin using 'helm plugin remove s3' and install again.
Error: plugin install hook for "s3" exited with error

I also see that there are no windows releases here:
https://github.com/hypnoglow/helm-s3/releases/

I use helm 2.11.0.

Thanks!
Guenael

**Comment by sweko** [comment]
It's possible to use helm-s3 on WSL, but I'd like to have a windows native version....

**Comment by ddezoysa** [comment]
Any progress on plugin support for Windows?

**Comment by hypnoglow** [comment]
Hey folks,

Support for Windows is merged. I have no Windows device at the moment, so I would appreciate if someone could test it. I've tagged [this commit](https://github.com/hypnoglow/helm-s3/pull/168) `v0.12.0-test.1` which includes Windows support.

Could you try and see if it works for you?

```
helm plugin install https://github.com/hypnoglow/helm-s3.git --version 0.12.0-test.1
```

If everything is ok, I'll prepare release version soon.

**Comment by hypnoglow** [comment]
Support for Windows should be available in [v0.12.0](https://github.com/hypnoglow/helm-s3/releases/tag/v0.12.0) release.
